pkg/plugin/plugin.go

```diff
@@ -102,7 +102,8 @@ func (p *plugin) GenerateConfig(ctx context.Context, cs *api.OpenShiftManagedClu
 
 func (p *plugin) GenerateARM(ctx context.Context, cs *api.OpenShiftManagedCluster, isUpdate bool) ([]byte, error) {
 	log.Info("generating arm templates")
-	return p.armGenerator.Generate(ctx, cs, isUpdate)
+	generator := arm.NewSimpleGenerator(p.entry)
+	return generator.Generate(ctx, cs, isUpdate, p.config)
```

Comment:
this change is due to a mishandled rebase - it shouldn't be here - see what `InitializeCluster` looks like below.